index.html

```diff
@@ -207,17 +207,20 @@ <h4>Skills</h4>
                         </ul>
                     </div>
                 </li>
-            </ul>
-            
+            </ul>        
+        </section>
+
+        <section id="contact-form">
             <div class="form-info">
                 <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
             </div>
 
-            <input type="text" class="input" placeholder="Full name">
-            <input type="email" class="input" placeholder="Email adress">
-            <textarea name="" id="" cols="30" rows="10" class="input textarea" placeholder="Enter text here"></textarea>
-            
-            <a href="#" class="button">Get in touch</a>
+            <form action="">
+                <input type="text" class="input" placeholder="Full name" maxlength="30" required>
+                <input type="email" class="input" placeholder="Email adress" required>
+                <textarea name="" id="" cols="30" rows="10" class="input textarea" placeholder="Enter text here" maxlength="500" required></textarea>
+                <input type="submit" id="submit-button" class="button" value="Get in touch">
```

Comment:
- [ ] I think it would be better to add the button's effects as presented in the Figma design for a good UX :recycle: 
![Screenshot from 2023-01-18 00-39-34](https://user-images.githubusercontent.com/97834160/213027968-34561acf-b08b-4f0e-865a-d2956b8d0178.png)

Reply:
That feature is already implemented in this pull request, you can see the css file and see that the :hover and :active button are already added and working, if you don't mean that let me know please